acceptance/acceptance_test.go

```diff
@@ -5,221 +5,91 @@ package acceptance
 import (
 	"bytes"
 	"context"
-	"crypto/sha256"
-	"encoding/hex"
 	"fmt"
-	"io"
 	"io/ioutil"
 	"math/rand"
 	"os"
-	"os/exec"
 	"path/filepath"
 	"runtime"
 	"strings"
 	"testing"
-	"text/template"
 	"time"
 
-	"github.com/Masterminds/semver"
-	dockertypes "github.com/docker/docker/api/types"
-	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/client"
-	"github.com/docker/go-connections/nat"
 	"github.com/google/go-containerregistry/pkg/name"
-	"github.com/pkg/errors"
 	"github.com/sclevine/spec"
 	"github.com/sclevine/spec/report"
 
-	"github.com/buildpacks/pack/internal/api"
-	"github.com/buildpacks/pack/internal/archive"
-	"github.com/buildpacks/pack/internal/blob"
-	"github.com/buildpacks/pack/internal/builder"
+	"github.com/buildpacks/pack/acceptance/assertions"
+	"github.com/buildpacks/pack/acceptance/components"
+	"github.com/buildpacks/pack/acceptance/managers"
 	"github.com/buildpacks/pack/internal/cache"
-	"github.com/buildpacks/pack/internal/logging"
 	"github.com/buildpacks/pack/internal/style"
 	h "github.com/buildpacks/pack/testhelpers"
 )
 
-const (
-	envCompilePackWithVersion = "COMPILE_PACK_WITH_VERSION"
-
-	runImage   = "pack-test/run"
-	buildImage = "pack-test/build"
-
-	defaultCompilePackVersion = "0.0.0"
-	defaultPlatformAPIVersion = "0.3"
-)
-
 var (
-	dockerCli      client.CommonAPIClient
-	registryConfig *h.TestRegistryConfig
-	suiteManager   *SuiteManager
+	dockerCli    *client.Client
+	suiteManager *managers.SuiteManager
 )
 
-type testWriter struct {
-	t *testing.T
-}
-
-func (w *testWriter) Write(p []byte) (n int, err error) {
-	w.t.Logf(string(p))
-	return len(p), nil
-}
-
 func TestAcceptance(t *testing.T) {
 	var err error
 
 	h.RequireDocker(t)
 	rand.Seed(time.Now().UTC().UnixNano())
 
 	dockerCli, err = client.NewClientWithOpts(client.FromEnv, client.WithVersion("1.38"))
```

Comment:
Don't we prefer to create a constant variable for the version (`1.38`)?
In this way we can have all of the constants in one place and if something is changed, we don't need to search for them in all of the file.

Reply:
Probably, but it's kind of not an interesting input to us. Pack doesn't care about using version 1.38, it just needs a docker client that works. It would be a change across a number of files outside of the acceptance test suite, so I wouldn't consider it part of the scope of this PR.